### ISSUE TITLE
Tests don't understand CSS variables

### ISSUE THREAD

**Issue by tolnem** [qualweb/wcag-techniques#3]
In the test WCAG-T28, having a font-size set to var(--font-size-tiny) will make the test fail, because there is "in" in the word tiny, regardless of what that variable is set to.

I created the page https://test.symbiotisk.dk/WCAG-T28.html as a proof of concept, and it currently fails the test, because of the variable name, and because the test currently doesn't actually check for important, as reported in another issue.
